system-variables.md

```diff
@@ -1554,6 +1554,16 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Default value: `tikv,tiflash,tidb`
 - This variable is used to set the storage engine list that TiDB can use when reading data.
 
+### tidb_last_query_info <span class="version-mark">New in v4.0.14</span>
+
+- Scope: SESSION
+- Default value: ""
```

Comment:
Does it persist to cluster?

Reply:
`SESSION` scope system variables never persist to cluster.